src/components/Button.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import Loader from './Loader';
-import { colors, fonts, shadows, transitions } from '../styles';
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import Loader from "./Loader";
+import { colors, fonts, shadows, transitions } from "../styles";
```

Comment:
lol someone got a new linting profile

Reply:
It was always @sudoryan that had a different linting profile ahah we need to enforce it using `prettierrc` file